kubernetes/api/v1alpha1/servicelevelobjective_types.go

```diff
@@ -123,6 +128,12 @@ type LatencyIndicator struct {
 	Grouping []string `json:"grouping"`
 }
 
+type BoolGaugeIndicator struct {
+	Query `json:",inline"`
```

Comment:
Not sure if we should inline it here. We don't do it on the other indicators. A YAML example should help.

Reply:
YAML example is here: examples/prometheus-probe-success.yaml